test/integration/pull_int_test.go

```diff
@@ -81,20 +85,16 @@ func (suite *PullIntegrationTestSuite) TestPull_Merge() {
 	ts.LoginAsPersistentUser()
 
 	cp := ts.SpawnWithOpts(e2e.WithArgs("push"), e2e.WithWorkDirectory(wd))
-	cp.ExpectLongString("Your project has new commits available")
+	cp.ExpectLongString("Your project has new changes available")
 	cp.ExpectExitCode(1)
 
 	cp = ts.SpawnWithOpts(e2e.WithArgs("pull"), e2e.WithWorkDirectory(wd))
 	cp.ExpectLongString("Merging history")
 	cp.ExpectExitCode(0)
 
 	cp = ts.SpawnCmd("bash", "-c", fmt.Sprintf("cd %s && %s history | head -n 10", wd, ts.ExecutablePath()))
-	cp.ExpectLongString(locale.T("pull_merge_commit"))
+	cp.ExpectLongString("Merged")
 	cp.ExpectExitCode(0)
-
-	cp = ts.SpawnWithOpts(e2e.WithArgs("push"), e2e.WithWorkDirectory(wd))
-	cp.ExpectLongString("You do not have permission") // This tells us the history is valid, we don't need an actual successful push
-	cp.ExpectExitCode(1)
```

Comment:
I'm not sure how this worked before... we've just pushed and it worked, we push again and expect it to fail?